### DISCUSSION TITLE
How to set the default selection in Select widget

### DISCUSSION THREAD

**Discussion by Tim-Cao** [Textualize/textual#3164] [Q&A] [answered]
I would like to set a default selection in Select widget.
For example,
The default one is LPS
```
                yield Label("Select the Project Key: ")
                yield Select([("LPS", "LPS"), ("LRQA", "LRQA"), ("LRAC", "LRAC"), ("COMMERCE", "COMMERCE")])
```

**Comment by davep** [comment, marked as answer]
See the `value` parameter in [the docs](https://textual.textualize.io/widgets/select/#textual.widgets.Select).

**Reply by Tim-Cao** [reply]
Like this?
```
yield Select([("LPS", "LPS"), ("LRQA", "LRQA"), ("LRAC", "LRAC"), ("COMMERCE", "COMMERCE")], value=("LPS", "LPS"))
```

**Reply by davep** [reply]
Not quite, `value` should be the value for the selection. In your case `value="LPS"` should be it.

**Reply by Tim-Cao** [reply]
I see. Many thanks
